internal/scorecard/helpers/helpers.go

```diff
@@ -132,47 +54,69 @@ func TestResultToScorecardTestResult(tr TestResult) scapiv1alpha1.ScorecardTestR
 	return sctr
 }
 
-// UpdateState updates the state of a TestResult.
-func UpdateState(res scapiv1alpha1.ScorecardTestResult) scapiv1alpha1.ScorecardTestResult {
-	if res.State == scapiv1alpha1.ErrorState {
-		return res
+// ResultsCumulative takes multiple TestResults and returns a single TestResult with MaximumPoints
+// equal to the sum of the MaximumPoints of the input and EarnedPoints as the sum of EarnedPoints
+// of the input
+func ResultsCumulative(results []TestResult) (TestResult, error) {
```

Comment:
Now that we've removed Maximum and Earned points, is this function still necessary? Is it possible to get rid of the `IsCumulative()` function on the `Test` and collapse this function with `ResultsPassFail`?

Lastly (and this may be in a part 3 PR), do we need to be combining results anymore?

Reply:
yeah, I was going to address cumulative in a 3rd PR given the scope and testing for that one...my hope is to get rid of merging/cumulative but I need to think about how to handle multiple CR processing/reporting probably at that same time.